pubsub/gcloud/aio/pubsub/metrics.py

```diff
@@ -35,6 +35,8 @@
         namespace=_NAMESPACE,
         subsystem=_SUBSYSTEM,
         unit='seconds',
+        buckets=(.005, .01, .025, .05, .075, .1, .25, .5, .75, 1.0, 2.5,
```

Comment:
Can probably trim off some of these smaller ones, that level of granularity is going to be dominated just in transit time anyway. Maybe just do something like:
```
buckets=(.01, .1, .25, .5, 1.0, 2.5, 5.0, 7.5, 10.0, 20.0, 30.0, float('inf'))
```

Reply:
Done!

Reply:
nit: while you're here, `float('inf')` is unnecessary

Reply:
Is it? I've seen it used in the prometheus libraries themselves as well

Reply:
Yeah we leave it out elsewhere and still receive values for those buckets. Interesting though, do you have a link? I only mention to avoid confusion in the places we've omitted including this, but it's fine if we want to keep it to be explicit here.

Reply:
The second link is the one I saw

Reply:
I believe they always add an inf bucket at the end if it's missing, so including it ourselves is a noop/stylistic choice.